src/pepr/operator/crd/sources/package/v1alpha1.ts

```diff
@@ -607,9 +607,23 @@ export const v1alpha1: V1CustomResourceDefinitionVersion = {
             authserviceClients: {
               type: "array",
               items: {
-                type: "string",
+                type: "object",
+                required: ["name", "clientId"],
+                properties: {
+                  name: { type: "string" },
```

Comment:
Is there a reason we have name here? I might've missed somewhere but I don't think it's actually used anywhere? In terms of whether we need it, we shouldn't since the `clientId` is the more unique identifier that all of our logic is based on.